src/providers/ldap/sdap_async_connection.c

```diff
@@ -1832,6 +1834,11 @@ static void sdap_cli_auth_step(struct tevent_req *req)
      */
     now = time(NULL);
     expire_timeout = dp_opt_get_int(state->opts->basic, SDAP_EXPIRE_TIMEOUT);
+    expire_offset = dp_opt_get_int(state->opts->basic, SDAP_EXPIRE_OFFSET);
```

Comment:
Would it make sense to check for negative value?

Or, put it other way, would behaviour in case of negative value meet user expectation?